### PR TITLE
Imp/1226 branding

### PR DESCRIPTION
**Pull request by jas29**
Solves issue #1226 

Added algolia branding to search

image

image

### CODE REVIEW

File: app-web/src/components/UI/AlgoliaBranding/AlgoliaBranding.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Aux from '../../../hoc/auxillary';
+import logo from '../../../assets/images/algolia.svg';
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+
+const noMargin = css`
```
Comment (patricksimonian) [inline app-web/src/components/UI/AlgoliaBranding/AlgoliaBranding.js]: using css and style  are probably not necessary at the same time. just move the margin bottom to styled.

File: app-web/src/pages/index.js [hunk 2]
```diff
@@ -381,6 +382,7 @@ export const Index = ({
         searchSourcesLoading={searchSourcesLoading}
         resultCount={totalSearchResults}
       />
+      <AlgoliaBranding />
```
Comment (patricksimonian) [inline app-web/src/pages/index.js]: let's remove the branding from now until algolia is integrated !

File: app-web/src/components/UI/AlgoliaBranding/AlgoliaBranding.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  margin-top: 15px;
+  height: auto;
+  width: 10%;
+  @media (max-width: 480px) {
+    display: none;
+  }
+`;
+
+const AlgoliaLogo = () => (
+  <Aux>
```
Comment (patricksimonian) [inline app-web/src/components/UI/AlgoliaBranding/AlgoliaBranding.js]: use a div so we have a  wrapper container

File: app-web/src/components/UI/AlgoliaBranding/AlgoliaBranding.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+`;
+const LargeLogo = styled.img`
+  ${noMargin}
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 15px;
+  height: auto;
+  width: 10%;
+  @media (max-width: 480px) {
```
Comment (patricksimonian) [inline app-web/src/components/UI/AlgoliaBranding/AlgoliaBranding.js]: shouldn't need a media query and width should be 100%

Comment (jas29) [reply app-web/src/components/UI/AlgoliaBranding/AlgoliaBranding.js]: Necessary css changes made

**Pull request merged**